### ISSUE TITLE
Verse bridge blocks Diglot-facing-pages

### ISSUE THREAD

**Issue by KimB2017** [sillsdev/ptx2pdf#845]
I'm trying the diglot on facing pages layout as summarized by David G on 16 January 23, in the message thread, "[SIL] [PTXprint] Diglot on Facing Pages (Single Column per page)".

PTXPrint failed with an error "missing space after verse number 3" in RTTT project. Turns out that both projects have a verse bridge here '\v 3-4', so the two versions' verses match, but PTXPrint doesn't seem to be able to handle this. Both projects passed the chapter,verse,marker checks.

**Comment by davidg-sil** [comment]
Hmm. 
I *assume* that you're not the first one to try to do a diglot  with bridged verses. Surely??  No, I can use the test files and make one that works:
![image](https://user-images.githubusercontent.com/58848232/227989256-402e4053-4f9c-4ce0-8ce6-1c6e9abc295c.png)
(That one had the bridge in the primary... is your text with bridged verses the primary or the secondary verse?)

My suspicion is that you do **not** have unicode character "U+002D HYPHEN-MINUS" between the numbers in the verse range, either because paratext or some other program 'helpfully' changed it for you, someone typed another character, or there is a changes file somewhere that changes it.
Might that be the case?

**Comment by KimB2017** [comment]
You are right, David. My PrintDraftChanges.txt file has (for years) changed the verse bridge hyphen u002D to u2013, which looks much better when printed and is a somewhat standard tweak, I think. 

Could the rule to change the bridge be applied after the basic diglot layout is complete? Or maybe we have to choose between diglot and pretty bridge-characters?

**Comment by davidg-sil** [comment]
I think that if you turn off that rule, you won't notice any difference, except that other things may work better.... The TeX code already notices bridged verses (if they are separated by u002D)  and prints them with an endash  as separator.
